bam_plcmd.c

```diff
@@ -1106,7 +1106,6 @@ int bam_mpileup(int argc, char *argv[])
             break;
         case  14:
             mplp.flag |=  MPLP_PRINT_QPOS5;
-            mplp.flag &= ~MPLP_PRINT_QPOS;
```

Comment:
You also need to remove the test on `MPLP_PRINT_QPOS5` on line [1104](https://github.com/samtools/samtools/blob/e56146ca/bam_plcmd.c#L1104)